CHANGELOG.md

```diff
@@ -20,6 +20,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
   - -Wno-<option>: Disable the corresponding warning.
   - -Werror: Turn warnings into errors.
   - -Werror=<option>: Turn the specified warning into an error.
+- `libsgx-quote-ex` and SGX AESM plugins will be installed on Linux starting from v0.15.0.
```

Comment:
@yentsanglee should we say in the change log that Linux will use out of proc attestation by default?

Reply:
> @yentsanglee should we say in the change log that Linux will use out of proc attestation by default?

Linux will not use out of proc attestation by default. Code path is controlled by SGX_AESM_ADDR environment variable.